docs/src/md/kotlin.core/syntax.md

```diff
@@ -622,6 +622,10 @@
   ~ `'0'` (`'b'` | `'B'`) [_BinDigit_](#grammar-rule-BinDigit) {[_BinDigitOrSeparator_](#grammar-rule-BinDigitOrSeparator)} [_BinDigit_](#grammar-rule-BinDigit)  
   | `'0'` (`'b'` | `'B'`) [_BinDigit_](#grammar-rule-BinDigit)
 :::
+:::{ .grammar-rule #grammar-rule-UnsignedLiteral }
+**_UnsignedLiteral_**  
+  ~ ([_IntegerLiteral_](#grammar-rule-IntegerLiteral) | [_HexLiteral_](#grammar-rule-HexLiteral)   | [_BinLiteral_](#grammar-rule-BinLiteral)) [`'L'`] (`'u'` | `'U'`)
```

Comment:
Thank you for your PR!
As one can see in the lexer code https://github.com/Kotlin/kotlin-spec/blob/develop/grammar/src/main/antlr/KotlinLexer.g4#L228, `L` is optional and always going after `U`